tests/testthat/test-encoding.R

```diff
@@ -57,9 +57,9 @@ test_that("Support non-ASCII strings not in UTF-8 encodings", {
   wb <- createWorkbook(creator = non_ascii[1])
   ws <- addWorksheet(wb, non_ascii[2])
   writeDataTable(wb, ws, non_ascii_df, tableName = non_ascii[3])
-  writeData(wb, ws, non_ascii_df, xy = list("D", 1), name = non_ascii[4])
+  writeData(wb, ws, non_ascii_df, xy = c(4, 1), name = non_ascii[4])
```

Comment:
why are these changes included? did something else change and now they bark?

Reply:
Changed back to the initial code